core/templates/dev/head/player/PlayerServices.js

```diff
@@ -315,6 +317,7 @@ oppia.factory('oppiaPlayerService', [
             }
           }
 
+          $location.path('/preview/' + newStateName);
```

Comment:
I don't think you can do this -- it messes up the learner view. Is it actually necessary?